static/js/tools/map/chart_loader.tsx

```diff
@@ -87,6 +93,7 @@ export function ChartLoader(): JSX.Element {
   useFetchGeoRaster(dispatchChartStore);
   useFetchAllDates(dispatchChartStore);
   useFetchStatVarSummary(dispatchChartStore);
+  useFetchBorderGeoJson(dispatchChartStore);
```

Comment:
should also update this ready hook so that in cases where we need the border, we only draw the map once the border is ready: https://github.com/datacommonsorg/website/blob/master/static/js/tools/map/ready_hooks.ts#L28

Reply:
Thanks for the catch! Updated the `useGeoJsonReady()` ready hook.